translations-script/export.js

```diff
@@ -4,39 +4,70 @@ const fs = require('fs');
 const _ = require('lodash');
 const Excel = require('exceljs');
 
-function process(ws, path, enObj, gaObj) {
-  if (!_.isObject(enObj)) {
-    const enVal = enObj.replace(/\n/g, '\\n');
-    const gaVal = _.get(gaObj, path, '');
+function process(ws, path, en, ht, ru, bn, ko, zh) {
+  if (!_.isObject(en)) {
+    const enVal = en.replace(/\n/g, '\\n');
```

Comment:
What is going on here with this object check on the en param only and the subsequent newline replaces. Why is this language singled out?